### ISSUE TITLE
`@stylistic/indentation`: false positive when `:where()` is not the first selector in a multi-selector rule

### ISSUE THREAD

**Issue by zipper** [stylelint-stylistic/stylelint-stylistic#74]
### What minimal example or steps are needed to reproduce the bug?

The following is correctly formatted CSS/SCSS (`indentation: 4`, top-level, no nesting):

```scss
.foo,
:where(
    .bar,
    .baz
) {
    margin-top: 0;
}
```

Errors reported with **v5.0.1**:
```
3:1  ✖  Expected indentation of 4 spaces  @stylistic/indentation
4:5  ✖  Expected indentation of 8 spaces  @stylistic/indentation
5:5  ✖  Expected indentation of 8 spaces  @stylistic/indentation
6:1  ✖  Expected indentation of 4 spaces  @stylistic/indentation
```

The rule expects `:where(` and its content to be indented by 4 spaces, and the inner selectors by 8 — as if `:where()` were nested inside the preceding `.foo` selector.

Same issue occurs inside SCSS nested rules:

```scss
.outer {
    :where(
        .foo,
        .bar
    ) > :first-child,
    :where(        /* ← error: expected 8 spaces */
        .foo,
        .baz
    ):last-child {
        margin-top: 0;
    }
}
```

And with a SCSS `#{}` interpolation as the first selector:

```scss
#{some-function()},
:where(    /* ← error: expected 4 spaces */
    .bar,  /* ← error: expected 8 spaces */
    .baz
) {
    margin-top: 0;
}
```

### What did you expect to happen?

No errors. The code is correctly formatted — `:where(` is at the correct nesting level and its content is indented exactly one level deeper.

### Versions

- `@stylistic/stylelint-plugin`: 5.0.1
- `stylelint`: 17.0.0

### Additional context

This is related to the fix in #30, but is a **different case**. The existing fix handles `:where()` as the first and only selector. This bug occurs when `:where()` is the **second or later** selector in a comma-separated list — the rule then incorrectly treats `:where()` as nested inside the preceding selector and shifts the expected indentation accordingly.

Reproduces with both plain CSS and SCSS (`customSyntax: postcss-scss`).

**Comment by firefoxic** [comment]
@zipper Hi! Thanks for the report!
The fix will be included in the next release.
